editor/src/clj/editor/gui.clj

```diff
@@ -336,7 +336,7 @@
   (-> (doto (Quat4d.)
         (math/clj->vecmath clj-quat))
       math/quat->euler
-      properties/round-vec
+      properties/round-vec-coarse
```

Comment:
Use coarse rounding when converting quaternions to euler angles for better value stability.

Prior to this PR, we used coarse rounding (three significant decimal places) everywhere. With this change, we use moderate rounding (six significant decimal spaces) for everything but the quaternion to euler angle conversion.